.github/workflows/feed.yml

```
@@ -82,3 +82,39 @@ jobs:
         type: 'stream'
         topic: 'New package registrations'
         content: ${{ steps.zulip.outputs.content }}
+        
+  update-twiter:
+      if: ${{ github.event.label.name == 'new package' && github.repository == github.event.pull_request.head.repo.full_name }}
+      runs-on: ubuntu-latest
+      steps:
+      - uses: julia-actions/setup-julia@latest
```

It's usually more secure to specify a version, rather than using `latest`

---

This is just copied from the other parts of the action, should we update it for all of them ?

---

Yes, it'd probably be better.  I think we can do
```suggestion
      - uses: julia-actions/setup-julia@ce353a9bc8bd2a581d0533eb1d0ffc6f60563eb0 # v1.7.1
```